components/collator/data/Cargo.toml

```diff
@@ -26,3 +26,4 @@ include = [
 ]
 
 [dependencies]
+icu_provider = { path = "../../../provider/core" } # For doc links
```

Comment:
The bake data exporter generates code like

```rust
/// Implement [`DataProvider<M>`](icu_provider::DataProvider) on the given struct
```

but `icu_provider` is not a dependency of the compiled data crates, so the doc link doesn't work.

Reply:
Can you clarify? There are no features on this crate.

Reply:
I would rather avoid the link; I think having data crates be non-leaf nodes is going to be a pain for people who are replacing them. I think it's fine for this "mostly internal" macro to just say `DataProvider`.

(It's definitely going to be a pain for google3)

Reply:
To be clear is your request to simply remove the docs link?

Reply:
Discussed: remove the links.